test/test_logger.rb

```diff
@@ -0,0 +1,169 @@
+# frozen_string_literal: true
+
+require_relative 'helper'
+require 'sidekiq/logger'
+
+class TesLogger < Minitest::Test
+  describe Sidekiq::Logger do
+    after do
+      Thread.current[:sidekiq_context] = nil
+    end
+
+    describe 'initialization' do
+      describe 'formatter' do
+        subject { Sidekiq::Logger.new(STDERR).formatter }
+
+        describe 'default formatter' do
+          it 'sets pretty formatter' do
+            assert_kind_of Sidekiq::Logger::PrettyFormatter, subject
+          end
+        end
+
+        describe 'when DYNO env var is present' do
+          before do
+            ENV['DYNO'] = 'dyno identifier'
```

Comment:
Looks ugly, but have no idea why the next doesn't work:
```
allow(ENV).to receive(:[]).with('DINO').and_return('dyno identifier')
```
Maybe because of the different string `'DINO'` objects?

Reply:
You've misspelled DYNO, that's why?

Reply:
Ah, shame to me, thank you ^_^ BTW, how it should look like in `minitest`? Like this:
```
ENV['DYNO'].stub do
  ...
end
```
?